packages/ckeditor5-package-generator/tests/index.js

```diff
@@ -99,89 +115,124 @@ describe( 'lib/index', () => {
 		expect( index ).to.be.a( 'function' );
 	} );
 
-	it( 'passes correct arguments to the validatePackageName()', async () => {
+	it( 'creates logger with verbose option set to false', async () => {
```

Comment:
Perhaps I do not understand this test, but it looks like we check something with the `validatePackageName()` function instead of the logger.

Reply:
Also, the description should answer the question "why?". Why does it create a logger with the verbose option set to false? It makes it by default when the option is not specified. So, I would rename the test case: `creates a logger with the verbose option set to false if it is not specified`.